contracts/script/deploy/router/config/example_config.json

```diff
@@ -1,5 +1,10 @@
 {
     "initialOwner": "0x0000000000000000000000000000000000000001",
-    "initialCertVerifier": "0x0000000000000000000000000000000000000002",
+    "initABNConfigs" : [
+      {
+        "0_blockNumber": 0,
+        "1_certVerifierAddress": "0x0000000000000000000000000000000000000002"
```

Comment:
can't you just do a mapping 
```
{ block_number: address }
```
without the keys like we figured out last time we can do?

Reply:
oh my bad I was remembering something austin had figured out, but it was probably for the toml config files.